gpu-operator/release-notes.rst

```diff
@@ -31,9 +31,128 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
+  550 and 535 branch drivers do not yet support this mode.
+
+  In previous versions, the ``useOpenKernelModules`` field specified the driver containers to install the NVIDIA Open GPU Kernel module driver. 
+  This field is now deprecated and will be removed in a future release.
+  If you were using the ``useOpenKernelModules`` field, it's recommended that you update your configuration to use the `kernelModuleType` field instead.   
+
+* Added support for Ubuntu 24.04.
+
+* Added support for NVIDIA HGX GB200 NVL and NVIDIA HGX B200.
+
+* Added support for the NVIDIA Data Center GPU Driver version 570.124.06. 
+
+* Added support for KubeVirt and OpenShift Virtualization with vGPU v18 for A30, A100, and H100 GPUs.
+
+* Added support for NVIDIA Network Operator v25.1.0.
+  Refer to :ref:`Support for GPUDirect RDMA` and :ref:`Support for GPUDirect Storage`.
+
+* Added support for OpenShift v4.18.
+
+* Added support for Containerd v2.0.
+
+* Added support for Kubernetes v1.32.
+  Note that the minimum supported Kubernetes versions has been updated to v1.29.
+
+* Added support for new MIG profiles with HGX B200.
+
+  * Added support for the following profiles:
+
+    * ``1g.23gb``
+    * ``1g.23gb+me``
+    * ``1g.45gb``
+    * ``2g.45gb``
+    * ``3g.90gb``
+    * ``4g.90gb``
+    * ``7g.180gb``
+
+  * Added an ``all-balanced`` profile creates the following GPU instances:
+
+    * ``1g.23gb`` :math:`\times` 2
+    * ``2g.45gb`` :math:`\times` 1
+    * ``3g.90gb`` :math:`\times` 1
+
+* Added support for new MIG profiles with HGX GB200.
+
+  * Added support for the following profiles:
+
+    * ``1g.24gb``
+    * ``1g.24gb+me``
+    * ``1g.47gb``
+    * ``2g.47gb``
+    * ``3g.95gb``
+    * ``4g.95gb``
+    * ``7g.189gb``
+
+  * Added an ``all-balanced`` profile creates the following GPU instances:
+
+    * ``1g.24gb`` :math:`\times` 2
+    * ``2g.47gb`` :math:`\times` 1
+    * ``3g.95gb`` :math:`\times` 1
+
+Improvements 
+------------
+
+* Improved security by removing unnecessary permissions in the GPU Operator ClusterRole.
+
+* Improved GPU Operator metrics to include a `operatorMetricsNamespace` field that sets the metrcis namespace to `gpu_operator`.
+
+* Improved error handling in Driver Manager for Kubernetes by adding pod watch permissions.
+
+.. _v25.3.0-fixed-issues:
+
+Fixed Issues
+------------
+
+* Removed default liveness probe from the GDS and GDRCopy containers of the driver-daemonset. 
+  Long response times of the `lsmod` commands were causing timeout errors in the probe and unnecessary restarts of the container, resulting in the DaemonSet being in a bad state.
+
+* Fixed an issue where the GPU Operator failed to create a valid DaemonSet name on OpenShift Container Platform when using 64 kernel page size.
+  Refer to Github `issue #1207 <https://github.com/NVIDIA/gpu-operator/issues/1207>`__ for more details.
+
+* Removed deprecated `operator.defaultRuntime`` parameter.
```

Comment:
I think this clarification is important. We didn't remove this parameter, we are just suppressing it as the field ignored by the operator
```suggestion
* Removed the deprecated `operator.defaultRuntime`` parameter from OpenShift web console
```